### PR TITLE
Add an overview of the new aggregation flow

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

File: draft-ietf-ppm-dap.md [hunk 1]
```diff
@@ -1060,69 +1060,149 @@ of relevant: issue #89]]
...
+   "candidate prefixes" and the output consists of a sequence of 0s and 1s,
+   each indicating whether the corresponding candidate is a prefix of the
+   measurement from which the input shares were generated.
+
+1. To verify that the output shares, when combined, correspond to a valid,
+   refined measurement, where validity is determined by the VDAF itself. For
+   example, the Prio3Sum variant of Prio3 ({{Section 7.4.2 of !VDAF}}) requires
+   that the output shares sum up to an integer in a specific range; for
+   Poplar1, the output shares are required to sum up to a one-hot vector (i.e.,
+   a vector that is non-zero in at most one position).
```
Comment (tgeoghegan) [inline draft-ietf-ppm-dap.md:1087]: nit:
```suggestion
   Poplar1, the output shares are required to sum up to a vector that is non-zero in at most one position.
```
We don't need to define the term "one-hot" unless we're going to use it elsewhere in this document.

File: draft-ietf-ppm-dap.md [hunk 2]
```diff
@@ -1060,69 +1060,149 @@ of relevant: issue #89]]
...
+
+In general, refinement and verification are not distinct computations, since
+for some VDAFs, verification only may be achieved implicitly as a result of the
+refinement process. We instead think of these as properties of the output
+shares themselves: if prepration succeeds, then the resulting output shares are
```
Comment (tgeoghegan) [inline draft-ietf-ppm-dap.md:1092]:
```suggestion
shares themselves: if preparation succeeds, then the resulting output shares are
```

File: draft-ietf-ppm-dap.md [hunk 3]
```diff
@@ -1060,69 +1060,149 @@ of relevant: issue #89]]
...
+
+* For a 2-round VDAF, the Leader sends its first-round prep-share to the
+  Helper, who replies with the first-round prep-message and its second-round
+  prep-share. In the next request, the Leader computes its second-round
+  prep-share locally, commits and sends the second-round prep-message.
+
+* In general, each request includes the Leader's prep share for the previous
+  round and/or the prep message for the current round; correspondingly, each
+  response consists of the prep message for the current round and the Helper's
+  prep message for the next round.
```
Comment (tgeoghegan) [inline draft-ietf-ppm-dap.md:1177]:
```suggestion
  prep share for the next round.
```

**Review by tgeoghegan — APPROVED** [review]
Seems sound to me. In particular I think the distinction between refinement and verification is helpful.

File: draft-ietf-ppm-dap.md [hunk 4]
```diff
@@ -1060,69 +1060,148 @@ of relevant: issue #89]]
...
+
+* In general, each request includes the Leader's prep share for the previous
+  round and/or the prep message for the current round; correspondingly, each
+  response consists of the prep message for the current round and the Helper's
+  prep share for the next round.
+
+The Aggregators proceed in this ping-ponging fashion until a step of the
+computation fails, indicating the report is invalid and should be rejected, or
+the last round is reached and each Aggregator computes an output share. All
+told there there are `ceil((ROUNDS+1)/2)` HTTP reuqests sent, where `ROUNDS` is
```
Comment (branlwyd) [inline draft-ietf-ppm-dap.md:1181]:
```suggestion
told there there are `ceil((ROUNDS+1)/2)` HTTP requests sent, where `ROUNDS` is
```

nit: typo

**Review by branlwyd — APPROVED** [review]

**Comment by branlwyd** [comment]
For consistency with the rest of the document (once PR#393 lands), you may wish to say "preparation message/share" rather than "prep message/share". (alternatively, I can fix this up in PR#393 to use "prep" instead of "preparation".)

**Comment by cjpatton** [comment]
> For consistency with the rest of the document (once PR#393 lands), you may wish to say "preparation message/share" rather than "prep message/share". (alternatively, I can fix this up in PR#393 to use "prep" instead of "preparation".)

I think I would prefer the shortened version -- "preparation" is a bit of a mouthful when reading through the spec.

**Pull request merged**